src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
+
+        Lottery lottery = new Lottery();
+        Ball ball = lottery.getRandomBall();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        System.out.println(ball);
+        System.out.println(ball1);
+        System.out.println(ball2);
```

Comment:
use for loop instead
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/f860d958-a242-4b5d-89f2-e8c5f192c4d0)